lib/index.js

```diff
@@ -388,6 +389,24 @@ MapboxGeocoder.prototype = {
     return this.options.proximity;
   },
 
+  /**
+   * Get the text to use as the search bar placeholder
+   * 
+   * If placeholder is provided in options, then use options.placeholder
+   * Otherwise, if language is provided in options, then use the localized string of the first language if available
+   * Otherwise use the default
+   */
+  getPlaceholderText: function(){
+    if (this.options.placeholder) return this.options.placeholder;
+    if (this.options.language){
+      var firstLanguage = this.options.language.split(",")[0];
+      var language = subtag.language(firstLanguage);
+      var localizedValue = localization.placeholder[language];
+      if (localizedValue)  return localizedValue;
+    }
+    return 'Search';
```

Comment:
you could also lookup 'en' in localization.placeholder, but I don't think it really matters.